lib/glimesh_web/live/chat_live/index.html.leex

```diff
@@ -10,16 +12,28 @@
                             <span>Welcome to chat! Follow the rules.</span>
                         </div>
                           <%= for chat_message <- @chat_messages do %>
-                            <div id="<%= chat_message.id %>" class="bubble you">
-                                <%= if @can_moderate and Glimesh.Chat.can_moderate?(@streamer, chat_message.user) === false do %>
-                                <i class="fas fa-times-circle" phx-click="timeout_user" phx-value-user="<%= chat_message.user.username %>"></i>
+                            <%=cond do %>
```

Comment:
Do you think we can shorten this to a CSS class that is added on mention? That way we don't have to duplicate the code above.

Reply:
slightly confused but im pretty sure I made the change.